src/current/v23.2/automatic-cpu-profiler.md

```diff
@@ -18,22 +18,14 @@ You can configure automatic CPU profile capture with the following [cluster sett
 
 Cluster Setting | Description | Default Value | Recommended Value
 ----------------|-------------|---------------|------------------
-`server.cpu_profile.enabled` | Indicates if the Automatic CPU Profiler is on or off. | `false` | 
-`server.cpu_profile.cpu_usage_combined_threshold` | The baseline value for when CPU profiles should be taken. Collect profiles from each node that meets threshold. | MAX integer, such as `9223372036854775807` | `80`
+`server.cpu_profile.cpu_usage_combined_threshold` | The baseline value for when CPU profiles should be taken. Collect profiles from each node that meets threshold. This setting enables and disables automatic cpu profiling. | MAX integer, such as `9223372036854775807` | `80`
```

Comment:
Do we need to specify that it's enabled when the value is greater than 0 and disabled at the 0 value?

Reply:
```
> select variable, default_value, description from crdb_internal.cluster_settings where variable =
                             -> 'server.cpu_profile.cpu_usage_combined_threshold';
                     variable                     |    default_value    |                                                                                                                                                      description
--------------------------------------------------+---------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  server.cpu_profile.cpu_usage_combined_threshold | 9223372036854775807 | a threshold beyond which if the combined cpu usage is above, then a cpu profile can be triggered. If a value over 100 is set, the profiler will never take a profile and conversely, if a valueof 0 is set, a profile will be taken every time the cpu profileinterval has passed or the provided usage is increasing
```

Updated description of `server.cpu_profile.cpu_usage_combined_threshold`